python/Sentinel1_Stats/Sentinel1_Stats.ipynb

```diff
@@ -0,0 +1,2372 @@
z+OsW+M0lroPn6aPxw7lcsPnckr957L87a9gl1+jXubkvf/+Rsxx+cQ1Tr7L3LYf94N7L7sOoyuvI4s+YHOS+dz2ngTmgGuhQsQthVxt0uM+QqZ25+gtNpg9qwRzS6WjPpc+wGpP0vg5sHdOKRpS8v6el1p8f/d+SrWf8jVLUiFwcWvv1bfM9fDLnuECZP6SStguB/RdLViG53g9QfOxsikCQspghMq1NbkmTo1SVBI0OxRpMYV+e2ChU2w/+UGpqeor8hh1xS3/9nZFJ5q0lzZjbBtZH8X919zEgsvP47kpG6+0ThjI+Z8jWDdELfedwvYFocv7ifIB4QxRbMgNM/8bBGm51j/Ty6NmSQPfeA6sn92mKA/F9W9x9tAw0/vfh4fWVhJYzjUOAtbGw/ZbbPYvQ1k7JnxSiXgKfNpX//jPeG7776bWq32tNuefPJJPvShD3Hvvffy5JNPEovF+MIXvgDAqlWruPPOO9m2bRs33XQTr3nNa5ARi9DJJ5/MLbfcwllnnfXvPJsox5iJShIMg/LZa3XZwPScVoiGwGz4+oFQStehhprAvd18POJA/kUPx0glUXEbZQnSe8udo6YSyJFeGgNR0/KoA5BedDtlGbhe1Dg8jj+Q0eG4hqcBTxG9YrQD/X9LIbfAV/WGrh9WkuDgYZzReb15voiqViOFITUYIhHXymt2ocOhrZQGfxyewzo03TlelAPFNNveeCuk3vYQ6YSpW+ekXA8jmWwr0DY3cqOhw9eRIhZCd6wyijUNWEskIvBVNM5RGZMslztgpojKT5gm/rqh9rFlNqk9kYgoQ2VSMDZFYqyGUW1CsYwsV2FkABJxZDKu6S6V7BB+ROPUQnjLag25ZR1i81r2vSqPiMfwV/bjD+Q0bsDRIXUV5UhV00VOz2LNVvGHuxCDfRGQrXW/JMLzMapeO0rSmo8YRtSS8Sg+mkqhknF++KXPIAKoLVMU31xh/LvLqX5umOpcktWxaT48dB8XrnuCZatmSJ44S3x7guGfBMikLqditqgNJMemsjymxxmYPCPH8OljfLceo8dMoUIDDAhjiv5zR/nuxlu1F1upau8n7tDsUzgzDSpbB0gfFOx9bZYwE+eWJ7ZQLSfYtX+Aeq8J3fl2VyJZr3fY5aIyN1musPaLHsmf7kU1mqy6dpLnPnYJaqSfgQdD3Fv6uOdzJ/LkdcfyxJ3rmJgscP4tf8zrf/wWsnfv1zlXIfiHl36JtXe9kbVfvAZZ0FEoOb+gWzfa2jtXqYRuwBKPa6Sz3+kKphoNhG0RJhTSAr8vMqhB40yqDQYe9HGma8ycM4JIJSkek+cfB+/DXjAR1QYyl2TiVetwRhei5g0m1lSJxK5pQifCT5SqeCMFhOPwhZV3tZ/RnWd8HmfBI7c/xN45RmYfZPc1aPQKEGDWDPIPT3HXDZ9iYYOBnzKwJ0vYC01UNo0xNY89vkB5hYFbEIQ9OVLbxrjtrq/x+s/cqpeBXIrTX/lzVqyaIUgr7KrS1R9BoJ/ZhQr99xic8vNX8bW1t7L3VXo9xDLbjTnch7o4LbEXq2JoBdx0UQkH5dik70gRpH5x8h6lRwCBj/m0r+B3HBB+Ro/uui5//ud/zoc+9KGn3b59+3ZOPfVUcjlNAH/eeedx8803A3D66afT1aVRmps3b6bZbFKt6jqyLVu2sG7df6z/o0gmCVf0aySrZZLdUWznMQlCzJqHaHgEvVmaQxn83qTOU7Z4ZCPGpxaQJiyVtRdj2REnMDh7pzEquhOOSsRQjo25UMN0O92QWspFVatascVi7VIbhMA5MKvpJCs17YWlknohX6zoZMdLbbUUXMxy5Q93oY5di+gutFGWmKZW/C1qyjYrUKgbEBSrUbNyiRjsa+9LRqhfFSGO1SKjpBWq/sXWiyKXiTyIqDY40IC0ttJvNDB6u1FBgL3QQDRcHa6Lx9qgpZaHqDxNZxlOTnWUmW1jJJNYxWbbKBJj01F42cPIZXRTgIFeDVqpN3RLwVQCvztFbWMvHBxr71+jyzVYLpyd0yCuMEQkEpiVJso0WfmdGkZXHvvQDM7YAotjWcLWDeq1oaIQTY0UbQHW2khoESlaAw3iSyTaChwpf6kM62iIN5LntHddTWG3r3mDf1ogty+ksKOKWbQwhOScx1/Fj0bX87613+KRE2+mMRJy6DxTp2ZsS4fXM2lUuULXbU9p2tJ0iu4nXfaP9vLW+3+fVbe9GTFvc/I5O3jrhbczfs8I2zx9v42kLlFTpsnIHT5eb4JmziQxLxEhzG1JYx+Kcca6Pdhpj4VjFcUT+zX4rQVYXATQEqaBrFZRtqG91qh2++KRbdx++7/w4098kvogWE1Fs0fgLfcwJx3SB0ySu6LuRxFo8B/e/zr6vxEjtxvmPuC3DU9z50Fd5meZVI/t1empaJ4AmsQljEBqsRjdTyiSMyGVlXFdLuf7yPFJhOuTfGoWUa7Rc/8M1WP6KDw8zWMeoGDmrAHGz85RXi81eMuyCLvSuoSuVCHzk934XUnIZ2j22Ey8bpN+hqREhJI3HDyL8uoExdUmhy9bS3I2ZGFDgq4zJ8nvgFXfcimdoBvS73zLx8k/MoXwfMxilbAQpclqDVZ99iC9j2omuVsf0nSMb8jOsuneyzjw/xk8dONWRh8bpHvdHFZTEcaMqP1rExVzyO2tU72/F1uYfPriT1JfnoEg1NEEx2bgAZfvlLdy7YW3E6zsB8siKCT0WheC9fT+2m8tCoGvzKd9hc9U66bfUJ7Ro7/vfe/jiiuuoLe392m3H3/88dx7772Mj48ThiFf+cpXOHz48C9970tf+hLHHHMM2ey/XW7w60UQrBvSC6rroiZndI/PRkOTAjSbMDapu6NYBlYtwJ5vEPTnCYe6ozCo9o5aOdEWmlW1cqdNN8ohybaVKtOa6jC1bTw6jUWc1b6mLhSOjUintSIPdA2m352kcdwI4Uhvp2tRqytSpKyFZUfIYdG2xlvHsMcX8Lri+EMFjdLu7
```

Comment:
I would avoid putting so much focus on "sharing with different uses" and "exposing as a service" . These are a feature that are poorly standardized at the moment (and VITO only). The basic reason of existence of UDP's (code reuse) should come first. In a further section the sharing and service aspects can be discussed, but I would keep that separate from the standardized basics

Reply:
What is a pUpdatedgraph?

Reply:
@m-mohr thank you for noticing 😅. It seems like it was a mistype while adding a comment. It seems to be fine in the PR/notebook itself.